xds/internal/client/rds_test.go

```diff
@@ -488,7 +471,7 @@ func TestRDSCaching(t *testing.T) {
 // to be invoked with an error once the watchExpiryTimer fires.
 func TestRDSWatchExpiryTimer(t *testing.T) {
 	oldWatchExpiryTimeout := defaultWatchExpiryTimeout
-	defaultWatchExpiryTimeout = 1 * time.Second
+	defaultWatchExpiryTimeout = 10 * time.Millisecond
```

Comment:
Nit: Is this value going to be sufficient? Why don't we leave it at 1 second?

Reply:
Reverted.

I was running this test -count 100, it took too long.